kolibri/plugins/facility/assets/src/views/SelectionBottomBar.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+
+  <BottomAppBar>
+    <span class="message">{{ selectedMessage }}</span>
+    <KButton
+      :disabled="$attrs.disabled || buttonsDisabled"
+      :text="coreString('confirmAction')"
+      :primary="true"
+      @click="$emit('click-confirm')"
+    />
+  </BottomAppBar>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import BottomAppBar from 'kolibri.coreVue.components.BottomAppBar';
+  import responsiveWindowMixin from 'kolibri.coreVue.mixins.responsiveWindowMixin';
+
+  export default {
+    name: 'SelectionBottomBar',
+    components: {
+      BottomAppBar,
+    },
+    mixins: [commonCoreStrings, responsiveWindowMixin],
+    props: {
+      counts: {
+        type: Number,
+        required: true,
+      },
+      type: {
+        type: String,
+        required: true,
+        validator(value) {
+          return value === 'learners' || value === 'coaches';
+        },
+      },
+    },
+    computed: {
+      buttonsDisabled() {
+        return this.counts === 0;
+      },
+      selectedMessage() {
+        return this.counts === 0
+          ? this.$tr('zeroSelectedMessage', { type: this.type })
+          : this.$tr('selectedMessage', { counts: this.counts, type: this.type });
+      },
+    },
+    $trs: {
+      zeroSelectedMessage: '0 {type} selected',
+      selectedMessage: '{counts} {type} selected',
```

Comment:
We'll need two separate localizations for coaches and learners since they need to be translated differently. Here is an example from a different component

https://github.com/learningequality/kolibri/blob/91c4f2ae06cd73e3d6ad24ae305f1877b7d53f4f/kolibri/plugins/device/assets/src/views/ManageContentPage/SelectionBottomBar.vue#L151-L153

So the coaches version would be something like
```
        '{count, number} {count, plural, one {coach} other {coaches}} selected',

```